lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -129,6 +129,7 @@ def buffers
       def self.create_later(attrs, storage_adapter)
         element = new(attrs)
         storage_adapter.buffers[:upsert][element.unique_identifier] = element
+        element
```

Comment:
I'd be very surprised if this actually changes the return type of this method; unless it was changed recently the value of any assignment expression in Ruby was always the right-hand-side regardless of how complicated it was.